backend/src/main/kotlin/org/loculus/backend/api/SubmissionTypes.kt

```diff
@@ -263,8 +271,17 @@ data class OriginalData<SequenceType>(
         description = "The key is the segment name, the value is the nucleotide sequence",
     )
     val unalignedNucleotideSequences: Map<SegmentName, SequenceType?>,
+    @Schema(
+        example = """{"raw_reads": [{"fileId": "f1le-uuId-asdf", "name": "myfile.fastaq"]}""",
+        description = "A map from file field names, to lists of files. The files can also have URLs.",
+    )
+    val files: Map<String, List<FilesType>>? = null,
 )
 
+typealias OriginalData<SequenceType> = OriginalDataInternal<SequenceType, FileIdAndName>
```

Comment:
I don't know much Kotlin but this looks strange to me, a generic type that is in fact more generic than it seems?

Reply:
sorry to bring this back up again - but I don't see where `OriginalData<SequenceType>` is used? Why is this required? oh apologies SequenceType is generic